BeagleBone_black/src/sensors/main.cpp

```diff
@@ -20,11 +20,14 @@
 
 #include "sensors/main.hpp"
 
+#include <math.h>
+
 #include "data/data.hpp"
 #include "sensors/imu_manager.hpp"
 #include "sensors/bms_manager.hpp"
 #include "sensors/proxi_manager.hpp"
 
+constexpr float kWheelRadius = 1.1;   // TODO(anyone) Get wheel radius for optical encoder
```

Comment:
A better guess will be 0.04 (assuming it's metres - add units comment as well)

Reply:
You could also just have a circumference so that you don't need to compute it every time.